packages/compat/src/resolver-transform.ts

```diff
@@ -320,3 +328,16 @@ function handleComponentHelper(
 
   resolver.resolveComponentHelper(locator, moduleName, param.loc, impliedBecause);
 }
+
+function handleDynamicHelper(param: ASTv1.Node, resolver: Resolver, moduleName: string): void {
+  switch (param.type) {
+    case 'StringLiteral':
+      resolver.resolveDynamicHelper({ type: 'literal', path: param.value }, moduleName, param.loc);
+      break;
+    case 'TextNode':
```

Comment:
Silly question, but how is it possible to pass a `TextNode` to the helper keyword? There is probably a legit use-case I'm missing, but I can't seem to create that scenario in AST-explorer :smile:.

Reply:
Maybe it's not? I was copying the earlier work on the component helper and that does have handling for TextNode, and I don't recall if that's because we found a case or because some typescript types implied it was possible.